File: term5/data-mining/list1/task0.py
import numpy as np

a = np.arange(1, 101)
b = np.arange(1, 101, 2)
c = np.linspace(-1, 1, num=201) * np.pi
d = c[c != 0]
e_ = np.sin(np.arange(1, 101))
e = np.where(e_ > 0, e_, 0)
A = np.arange(1, 101).reshape(10, 10)
B = np.diag(np.arange(1, 101)) + np.diag(np.arange(99, 0, -1), k=1) + np.diag(np.arange(99, 0, -1), k=-1)
C = np.triu(np.ones(10))
D = np.vstack((np.cumsum(a), np.cumprod(a)))
E = np.where(np.atleast_2d(np.arange(1, 11)) % np.atleast_2d(np.arange(1, 11)).T > 0, 1, 0)

File: Eksploracja_danych/list2/task0.m
a = 1:100;
b = 1:2:100;
c = (-1:0.01:1) * pi;
d = [c(1:100), c(102:201)];
e = max(sin(1:100), 0);
A = reshape(1:100, [10, 10])';
B = diag(1:100) + diag(99:-1:1, -1) + diag(99:-1:1, 1);
C = triu(ones(10, 10));
D = [cumsum(1:100); cumprod(1:100)];
E = mod(1:100, (1:100)') == 0;
